sources/ghostdog/src/main.rs

```diff
@@ -117,7 +117,12 @@ fn parse_device_name(device_info: &[u8], path: String) -> Result<String> {
     let offset = NVME_IDENTIFY_DATA_SIZE - 1024;
     let device_name = &device_info[offset..offset + 32];
 
-    Ok(String::from_utf8_lossy(device_name).trim_end().to_string())
+    // Remove `/dev` in case the returned device name includes it, the udev
+    // rule already includes that prefix
+    Ok(String::from_utf8_lossy(device_name)
+        .replace("/dev/", "")
+        .trim_end()
+        .to_string())
```

Comment:
nit: it feels a bit loose not to anchor this to the start of the string:
```suggestion
    Ok(String::from_utf8_lossy(device_name)
        .trim_start_matches("/dev/")
        .trim_end()
        .to_string())
```